api/app/models/category.rb

```diff
@@ -0,0 +1,4 @@
+class Category < ApplicationRecord
+    validates :name, presence: true, length: { maximum: 20 }
```

Comment:
```suggestion
    validates :name, presence: true, length: { maximum: 20 }, uniqueness: true
```
https://railsguides.jp/active_record_validations.html#uniqueness

Reply:
categoryの名前はunique